src/components/index/createRequestsTableCells.tsx

```diff
@@ -97,6 +102,9 @@ function createRequestsTableCells(
         req.requestTx || req.proposeTx || req.disputeTx || req.settleTx
       }&chainId=${req.chainId}&oracleType=${req.oracleType}&eventIndex=${0}`;
 
+      const eventBased =
+        req.eventBased === false ? "No" : req.eventBased === true ? "Yes" : "-";
+
```

Comment:
That's the 3rd value to this? Is it sometimes undefined / null ?

Reply:
I'd guess this is for loading state